qt/python/mantidqt/CMakeLists.txt

```diff
@@ -59,5 +59,7 @@ mtd_add_sip_module (
     ${PYTHON_LIBRARIES}
     ${Boost_LIBRARIES}
     API
+  INSTALL_DIR
+    ${LIB_DIR}/mantidqt
```

Comment:
This brings in `_commonqt5.so` to the MacOS bundle. Can you add an `if` so that it only installs on Linux?

Reply:
Oh. I thought I had managed to disabled that with https://github.com/mantidproject/mantid/pull/23211/files/800f7df7998fdb8ff80e13fe410d8037a7fc06cf#diff-f36073e58ea48a6c694c553145a7f17aR241. I'll take a look.

Does it do this even if `PACKAGE_WORKBENCH=OFF`?

Reply:
`_commonqt5.so` isn't copied when `PACKAGE_WORKBENCH=OFF`

Reply:
Correct